lib/Screens/AuthRegistration/body.dart

```diff
@@ -73,25 +74,55 @@ class _BodyState extends State<Body> {
           child: SingleChildScrollView(
             padding: EdgeInsets.symmetric(
               horizontal: 40,
-              vertical: 90,
+              vertical: 50,
             ),
             child: Column(
               children: [
-                Header('Sign In'),
-                Space(30),
+                Header('Sign Up'),
+                Space(20),
+                _buildInputFrame(
+                  label: 'Full Name',
+                  inputHint: 'Enter your Name',
+                  shouldInputBeObscured: false,
+                  inputType: TextInputType.name,
+                  prefixIcon: Icons.account_circle_rounded,
+                ),
+                Space(20),
+                _buildInputFrame(
+                  label: 'Phone No',
+                  inputHint: 'Enter your Phone No',
+                  shouldInputBeObscured: false,
+                  inputType: TextInputType.phone,
+                  prefixIcon: Icons.phone_android,
+                ),
+                Space(20),
                 _buildInputFrame(
                   label: 'Email',
                   inputHint: 'Enter your Email',
                   prefixIcon: Icons.email,
+                  inputType: TextInputType.emailAddress,
                   shouldInputBeObscured: false,
                 ),
-                Space(30),
+                Space(20),
                 _buildInputFrame(
                   label: 'Password',
                   inputHint: 'Enter your Password',
                   shouldInputBeObscured: true,
+                  inputType: TextInputType.visiblePassword,
                   prefixIcon: Icons.lock,
-                )
+                ),
+                Space(20),
```

Comment:
I checked your **Space** widget, and it takes one parameter and return SizedBox with the same parameter. Why don't we use just SizedBox, to be more clear ? :)

Reply:
Refactoring done! 👍